packages/client/src/aiClient.ts

```diff
@@ -17,7 +17,11 @@ export type Callbacks = {
   onComplete(): void;
   onRequestContext?: (
     data: Record<string, unknown>
-  ) => Record<string, unknown> | Promise<Record<string, unknown>>;
+  ) =>
+    | undefined
+    | Record<string, unknown>
+    | unknown[]
+    | Promise<Record<string, unknown> | unknown[] | undefined>;
```

Comment:
The backend (maybe incorrectly) already considers cases where the context is an `unknown[]`, but `undefined` is new. Is `undefined` necessary?

Reply:
You’re right we can return an empty object or array.